include/ofi_lock.h

```diff
@@ -161,13 +161,17 @@ static inline void ofi_fastlock_acquire_noop(fastlock_t *lock)
 	/* These non-op routines must be used only by a single-threaded code*/
 	assert(!lock->in_use);
 	lock->in_use = 1;
+#else
+    (void)lock;
 #endif
 }
 static inline void ofi_fastlock_release_noop(fastlock_t *lock)
 {
 #if ENABLE_DEBUG
 	assert(lock->in_use);
 	lock->in_use = 0;
+#else
+    (void)lock;
```

Comment:
Here and above please use tabs for indentation instead of spaces.  Also, while making that change, please add a space after the (void) before lock.

Reply:
Fixed